packages/slinkity-renderer-svelte/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       },
       plugins: [
         svelte({
-          preprocess: autoPreprocess,
```

Comment:
Noticed we weren't calling() this function, which broke `scss` support. Little fix for that here